src/App.vue

```diff
@@ -1364,6 +1365,11 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     ) {
       this.setShowErrors(true)
     }
+
+    // enable validation right away if user is changing a submitted filing
+    if (this.getFilingStatus === FilingStatus.CHANGE_REQUESTED) {
+      this.setShowErrors(true)
+    }
```

Comment:
In this scenario, the application should be fully valid, except we're going to make the Continuation Authorization files section invalid to bring the user's attention to the fact they need to make changes to their file(s) and we want to bring their attention to this. Ie,

![image](https://github.com/user-attachments/assets/6cf8c509-56f3-4b96-8e99-965747540bc2)